snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb

```diff
@@ -0,0 +1,112 @@
+Imports System.Text
+Imports System.Net.Sockets
+Imports System.Net.Security
+Imports System.Net
+'<snippet0>
```

Comment:
this line should be moved to precede `Imports System.Text` -- we want the list of imports to be included in the example.